packages/cosmic-swingset/lib/ag-solo/vats/bootstrap.js

```diff
@@ -277,7 +277,7 @@ export function buildRootObject(vatPowers, vatParameters) {
             },
             want: {
               // We just throw away our governance tokens.
-              Governance: amountMath.make(0n, govBrand),
+              Bld: amountMath.make(0n, govBrand),
```

Comment:
Would be capitalized.  BUT!  Governance is actaully the bank liquidity token, and is NOT  BLD.